tools/ci_build/github/linux/docker/scripts/install_openvino.sh

```diff
@@ -7,16 +7,16 @@ o) OPENVINO_VERSION=${OPTARG};;
 esac
 done
 
-OPENVINO_VERSION=${OPENVINO_VERSION:=2019_R1.1}
-git clone https://github.com/opencv/dldt.git /data/dldt/openvino_2019.1.144
+OPENVINO_VERSION=${OPENVINO_VERSION:=2019_R3.1}
+git clone https://github.com/opencv/dldt.git /data/dldt/openvino_2019.3.376
```

Comment:
is this git clone still needed? it looks like you aren't supporting build openvino from source anymore?

Reply:
Ok, I see. this is for the CI. we support building from source for CI?
But not with the build.py script? is that the distinction?

Reply:
Yes, for CI we try to build from opensource . But for build.py script we do it with binary to reduce the build time and also the footprint. To build dldt it takes sometime and also it does occupy some more space on the system

Reply:
That's right. Building from source is only supported for CI, which is why the option was removed from the build.py script.